curvefs/test/metaserver/metastore_test.cpp

```diff
@@ -94,7 +94,10 @@ class MetastoreTest : public ::testing::Test {
                    first.partitionid() == second.partitionid() &&
                    first.start() == second.start() &&
                    first.end() == second.end() &&
-                   first.nextid() == second.nextid();
+                   first.nextid() == second.nextid() &&
```

Comment:
you can use `google::protobuf::util::MessageDifferencer::Equals` to simplify this compare